files/bash_aliases

```diff
@@ -37,9 +41,28 @@ alias apacherestart='sudo httpd -e info -k restart'
 alias memcached-start='memcached -I2m -d start'
 alias mariastart='brew services start mysql'
 alias mariastop='brew services stop mysql'
+alias apache-logs='cd /usr/local/var/log/httpd'
+alias codecept='php ~/work/testing-toolkit/vendor/codeception/codeception/codecept'
 
 ## OPEN FILE WITH TEXTWRANGLER
 function textWranglerOpen() {
 	open -a TextWrangler $1
 }
 alias edit=textWranglerOpen
+
+## Misc
+goodMorningClosure() {
+	say "Good morning motherfucker!"
+	open -a "Google Chrome" https://mail.google.com/ https://github.com
```

Comment:
I really like this set of features! 
I initially considered that you make this configurable - but really when you are just starting off github and gmail is just enough. When the user is comfortable enough, he can tweak what to open.

To go even a step further, do you think it is possible to open these applications (slack, chrome, etc.) in fullscreen mode and define their relative order. For example first window is chrome, then terminal, then phpstorm

Reply:
Agreed